tests/salt_nanny_client_test.py

```diff
@@ -10,9 +10,20 @@ class SaltNannyClientTest(unittest.TestCase):
     def setup_client(self):
         client = SaltRedisClient('localhost', 6379, '0')
         client.redis_instance = FakeRedis()
+        client.redis_instance.flushall()
```

Comment:
You could use a setup instead to create a new FakeRedis each time (instead of reusing the same object and flushing it)